### DISCUSSION TITLE
DIVERGED_PC_FAILED

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#27412] [Q&A Modules: Phase field]
I am trying to run a phase field simulation coupled with crystal plasticity and below is a snippet of the error I am getting:
Time Step 1, time = 0.0005, dt = 0.0005
    |residual|_2 of individual variables:
                     c:      0
                     eta1:   0.000389647
                     eta3:   0.622528
                     c1:     107.533
                     c3:     0.151691
                     disp_x: 30.3519
                     disp_y: 17.3831
                     disp_z: 55.1197
 0 Nonlinear |R| = [32m1.257985e+02[39m
  Linear solve did not converge due to DIVERGED_PC_FAILED iterations 0
                 PC failed due to SUBPC_ERROR 

In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

A MooseException was raised during FEProblemBase::computeResidualTags
In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

Nonlinear solve did not converge due to DIVERGED_LINE_SEARCH iterations 0
[31m Solve Did NOT Converge![39m
Aborting as solve did not converge

Solve failed, cutting timestep.

Time Step 1, time = 0.00025, dt = 0.00025
    |residual|_2 of individual variables:
                     c:      0
                     eta1:   0.000389647
                     eta3:   0.622528
                     c1:     107.533
                     c3:     0.151691
                     disp_x: 15.174
                     disp_y: 8.69048
                     disp_z: 27.5495
 0 Nonlinear |R| = [32m1.123763e+02[39m
  Linear solve did not converge due to DIVERGED_PC_FAILED iterations 0
                 PC failed due to SUBPC_ERROR 

In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

A MooseException was raised during FEProblemBase::computeResidualTags
In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

Nonlinear solve did not converge due to DIVERGED_LINE_SEARCH iterations 0
[31m Solve Did NOT Converge![39m
Aborting as solve did not converge

Solve failed, cutting timestep.

Time Step 1, time = 0.000125, dt = 0.000125
    |residual|_2 of individual variables:
                     c:      0
                     eta1:   0.000389647
                     eta3:   0.622528
                     c1:     107.533
                     c3:     0.151691
                     disp_x: 7.58654
                     disp_y: 4.34497
                     disp_z: 13.7722
 0 Nonlinear |R| = [32m1.087648e+02[39m
  Linear solve did not converge due to DIVERGED_PC_FAILED iterations 0
                 PC failed due to SUBPC_ERROR 

In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

A MooseException was raised during FEProblemBase::computeResidualTags
In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

Nonlinear solve did not converge due to DIVERGED_LINE_SEARCH iterations 0
[31m Solve Did NOT Converge![39m
Aborting as solve did not converge

Solve failed, cutting timestep.

Time Step 1, time = 6.25e-05, dt = 6.25e-05
    |residual|_2 of individual variables:
                     c:      0
                     eta1:   0.000389647
                     eta3:   0.622528
                     c1:     107.533
                     c3:     0.151691
                     disp_x: 3.79315
                     disp_y: 2.17242
                     disp_z: 6.88544
 0 Nonlinear |R| = [32m1.078433e+02[39m
  Linear solve did not converge due to DIVERGED_PC_FAILED iterations 0
                 PC failed due to SUBPC_ERROR 

In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

A MooseException was raised during FEProblemBase::computeResidualTags
In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.

Nonlinear solve did not converge due to DIVERGED_LINE_SEARCH iterations 0
[31m Solve Did NOT Converge![39m
Aborting as solve did not converge

Solve failed, cutting timestep.

Time Step 1, time = 3.125e-05, dt = 3.125e-05
    |residual|_2 of individual variables:
                     c:      0
                     eta1:   0.000389647
                     eta3:   0.622528
                     c1:     107.533
                     c3:     0.151691
                     disp_x: 1.89655
                     disp_y: 1.08619
                     disp_z: 3.44256
 0 Nonlinear |R| = [32m1.076118e+02[39m
  Linear solve did not converge due to DIVERGED_PC_FAILED iterations 0
                 PC failed due to SUBPC_ERROR 

In computing the eigenvalues and eigenvectors of a symmetric rank-2 tensor, the PETSC LAPACK syev routine returned error code 2
To recover, the solution will fail and then be re-attempted with a reduced time step.
@GiudGiud @dschwen

**Comment by GiudGiud** [comment]
Hello

Do you know which rank two tensor the error message is mentioning? 
Can you simplify the problem to debug? Do you need to be solving for displacements and the c_i at the same time or can you only solve on equation to start with?

Guillaume
